src/app/shared/enum/provider.ts

```diff
@@ -58,4 +58,24 @@ export enum InstitutionTypes {
   Complex = 'Комплексний',
   Profile = 'Профільний',
   Specialized = 'Спеціалізований'
+}
+
+export enum PayRateType {
+  Classes,
+  Month,
+  Day,
+  Year,
+  Hour,
+  Course,
+  AllPeriod
+}
+
+export enum PayRateTypeUkr {
+  Classes = 'Заняття',
+  Month = 'Місяць',
+  Day = 'День',
+  Year = 'Рік',
+  Hour = 'Година',
+  Course = 'Курс',
+  AllPeriod = 'Увесь період'
```

Comment:
move this enum to the ukr folder (we store all Ukrainians values in the separate folder)